### DISCUSSION TITLE
Git LFS limit reached.

### DISCUSSION THREAD

**Discussion by haixuanTao** [webonnx/wonnx#64] [General]
As Git LFS limit has been reached and this is blocking for the CI/CD, we can either choose to pay or move to a different provider than Github.

I know that some people use Dropbox.

I think that moving to another provider will allow us to put more model and data and distribute it with more bandwith.  

I don't have any fixed solution in mind.

I am ok to pay for whatever is necessary but I think that a free option might be preferable.

**Comment by pixelspark** [comment]
Can we just do without Git LFS? The included models are only a few MBs..

**Comment by haixuanTao** [comment]
I think that it is going to depend on what the CI should cover. Let say we want to cover basic run of BERT in the future on all APIs E2E, it would be handy to have it somewhere.

**Comment by haixuanTao** [comment]
I guess AWS s3 Free Tier is also compelling:https://aws.amazon.com/s3/pricing/?loc=ft#AWS_Free_Tier
